pkg/dataset/priority_queue.go

```diff
@@ -89,3 +102,72 @@ func (pq *PriorityQueue) Pop() interface{} {
 	pq.rows = old.rows[0 : n-1]
 	return item
 }
+
+func (pq *PriorityQueue) update() {
+	heap.Fix(pq, pq.Len()-1)
+}
+
+func (value *OrderByValue) Compare(compareVal *OrderByValue, orderByItems []OrderByItem) int {
+	for _, item := range orderByItems {
+		compare := compareTo(value.OrderValues[item.Column], compareVal.OrderValues[item.Column], item.Desc)
+		if compare == 0 {
+			continue
+		}
+		return compare
+	}
+	return 0
+}
+
+func compareTo(a, b interface{}, desc bool) int {
+	if a == nil && b == nil {
+		return 0
+	}
+	if a == nil {
+		return 1
+	}
+	if b == nil {
+		return -1
+	}
+	// TODO Deal with case sensitive.
+	var (
+		result = 0
+	)
+	switch a.(type) {
+	case string:
+		result = compareValue(fmt.Sprintf("%v", a), fmt.Sprintf("%v", b))
+	case int8, int16, int32, int64:
+		result = compareValue(a.(int64), b.(int64))
```

Comment:
Should be `int64(xxx)`.

Reply:
> Should be `int64(xxx)`.

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/5917359/176632908-9718504f-e9bb-4116-a8df-f011802cd95a.png">

How to convert?